core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -72,10 +72,11 @@ object ConsoleConsumer extends Logging {
         new OldConsumer(conf.filterSpec, props)
       } else {
         val timeoutMs = if (conf.timeoutMs >= 0) conf.timeoutMs else Long.MaxValue
+        val consumer = new KafkaConsumer[Array[Byte], Array[Byte]](getNewConsumerProps(conf))
```

Comment:
Since we are overriding the deserializers anyway, maybe we could use the constructor which accepts the deserializers:
```scala
val consumer = new KafkaConsumer(getNewConsumerProps(conf), new ByteArrayDeserializer, new ByteArrayDeserializer)
```
We could then update `getNewConsumerProps` to get rid of the overrides and perhaps log a warning if the user has provided their own deserializer.